zk.js/src/build-balance.ts

```diff
@@ -212,7 +216,7 @@ export async function decryptAddUtxoToBalance({
 
   // null if utxo did not decrypt -> return nothing and continue
   if (!decryptedUtxo.value || decryptedUtxo.error) return;
-  if (decryptedUtxo.value && decryptedUtxo.value.utxoDataHash !== "0") return;
+  // if (decryptedUtxo.value && decryptedUtxo.value.dataHash !== "0") return;
```

Comment:
commented on purpose?

Reply:
I believe the reason for this check was to ensure that we don't decrypt program utxos and add them twice to the balance, once as normal utxo and once as program utxo.

Reply:
should not be needed anymore since program-utxo and utxo get encoded differently and programutxos should be able to go through as regular utxos at decryption/decoding i think